docs/source/torch.rst

```diff
@@ -79,6 +79,7 @@ Creation Ops
     complex
     polar
     heaviside
+    from_dlpack
```

Comment:
It'd be good to move this up, to in between `from_numpy` and `frombuffer`, since those are the two related functions and this one is more important than `frombuffer`.